lib/init.g

```diff
@@ -259,7 +241,9 @@ CallAndInstallPostRestore( function()
 end);
 
 ReadOrComplete := function(name)
-    InfoRead1( "#I  reading ", name, "\n" );
+    if GAPInfo.CommandLineOptions.D then
+        Print( "#I  reading ", name, "\n" );
+    fi;
```

Comment:
I tried introducing an `InfoClass` for this purpose, but as you say the infrastructure is not initialised when we'd want to use it.

One could (independently of this PR) try moving the info infrastructure as early as possible in startup.

Reply:
I think it is fine to leave this with a `Print` statement, but perhaps add a comment like
> We cannot use `Info` yet, because info.gi has not yet been read